setup.py

```diff
@@ -173,6 +179,7 @@ class Extension(extension.Extension, object):
     packages=['minorminer',
               'minorminer.layout',
               'minorminer.utils',
+              'minorminer.rpack',
```

Comment:
Since we're including rectangle-packer here verbatim (right?), I wouldn't expose it publicly like this. I'd keep it under `minorminer._vendor.rpack`, or similar. (`_vendor` namespace is used by Pip, but I've seen this handled differently by various packages over the years. E.g. `requests` used to pack a modified version of `urllib3`, etc.)

Reply:
I am personally a fan of `extern/`, it's what we use in [dimod](https://github.com/dwavesystems/dimod). But flexible on the name. Some way of signalling that it's external to this package would be good.